packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -973,6 +998,27 @@ class MultiSelectComboBox extends ResizeMixin(InputControlMixin(ThemableMixin(El
     this._overflowItems = items;
   }
 
+  /** @private */
+  __getExtraWidth() {
+    // Check if max-width is set on the host
+    const maxWidth = parseInt(getComputedStyle(this).maxWidth);
+    if (!isNaN(maxWidth)) {
+      return maxWidth - this.clientWidth;
+    }
+
+    // Check if width is set using style attribute
+    const width = parseInt(this.style.width);
+    if (!isNaN(width)) {
+      return width - this.clientWidth;
+    }
+
+    // Check if parent element allows to expand
+    const parent = this.parentNode;
+    const host = parent instanceof ShadowRoot ? parent.host : parent;
+
+    return host.clientWidth - this.clientWidth;
```

Comment:
`host.clientWidth` only works if the host doesn't contain other elements/padding or such that might limit the available horizontal space

Reply:
Removed this logic.